Added "permanently_closed" attribute to Spot

5290charlie opened the pull request:
The Google Places service returns "permanently_closed" if a place is considered permanently closed (nil value if the place is not closed). 
The current version of google_places does not support this attribute, I have added it to the GooglePlaces::Spot class

qpowell commented:
@5290charlie Thanks! :+1: . Would you mind adding the attribute to `spec/google_places/spot_spec.rb` as well ? Specifically in the 'Find a single spot' context.

Now that I'm maintaining this I need to do an audit soon to see what has changed in the api since the last time the gem was updated. Feel free to create an issue if you come across anything else that's missing :)

5290charlie commented:
I added the attribute to that spec, but 3 other specs are failing:
- rspec ./spec/google_places/spot_spec.rb:106 # GooglePlaces::Spot Multiple page request should return >20 results when :multipage_request is true
- rspec ./spec/google_places/spot_spec.rb:121 # GooglePlaces::Spot Multiple page request should return a pagetoken when there is more than 20 results and :multipage is false
- rspec ./spec/google_places/spot_spec.rb:151 # GooglePlaces::Spot List spots by radar should be a collection of Spots

It seems to have also modified the VCR cassettes. I'm still fairly new to Ruby (and rspec), are these failures due to the changes with the cassettes?

5290charlie commented:
I have added the new attribute to the spot_spec, but I'm still seeing two failures related to the Spot Multiple page request testing:

- rspec ./spec/google_places/spot_spec.rb:106 # GooglePlaces::Spot Multiple page request should return >20 results when :multipage_request is true
- rspec ./spec/google_places/spot_spec.rb:121 # GooglePlaces::Spot Multiple page request should return a pagetoken when there is more than 20 results and :multipage is false

Looks like maybe whatever responses were recorded by VCR have changed since the test(s) were originally written? I'm not sure, but that's what it looks like to me.

qpowell commented:
@5290charlie I'll take a look and see what's going on. Thanks

edwinwills commented:
Hi @5290charlie - I've pushed a few updates to the gem that should mean specs are passing 100% on master - if you want to rebase off the current master, this PR should hopefully be green and we can get it merged - cheers

edwinwills commented:
Closing, as updated with https://github.com/qpowell/google_places/pull/94
